components/adblock_rust_ffi/src/wrapper.h

```diff
@@ -5,8 +5,12 @@
 
 #ifndef BRAVE_COMPONENTS_ADBLOCK_RUST_FFI_SRC_WRAPPER_H_
 #define BRAVE_COMPONENTS_ADBLOCK_RUST_FFI_SRC_WRAPPER_H_
+
+#include <optional>
```

Comment:
I had them grouped together and then sorted as per `clang-format`, but `lint` was complaining about the order of C vs. C++ headers being wrong. This was the only way I found to satisfy both.

Reply:
Aha! This is something like a linter bug, since `<optional>` is definitely a C++ header.

However,  std::optional is [still banned](https://chromium.googlesource.com/chromium/src/+/102.0.5005.61/styleguide/c++/c++-features.md#std_optional-banned) in chromium code, based on hardening concerns. Best to replace this with absl::optional, and then the include order will make sense. :upside_down_face: